python/type_check_error.py

```diff
@@ -44,6 +44,12 @@ def display_type(n_type):
 	elif isinstance(n_type, tuple):
 		return Fore.YELLOW + ' -> '.join(n_type) + Style.RESET_ALL
 	elif isinstance(n_type, list):
+		try:
+			if(type(n_type[0]) == lark.Token):
+				if (n_type[0].type == "LIST"):
+					return Fore.YELLOW + '[' + ', '.join([str(n) for n in n_type[1:]]) + ']' + Style.RESET_ALL
```

Comment:
Perhaps instead of using `str(n)`, it should use `display_type` to allow for nested types, like a list containing lists. Also, shouldn't the displayed type for a list be `list[ ]` not `[ , , ]`?

Reply:
like sure i guess, i'll work on using display_type